src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -87,12 +87,12 @@ class MSUnmerged(MSCore):
     """
 
     # @profile
-    def __init__(self, msConfig, logger=None):
+    def __init__(self, msConfig, logger=None, mockMongoDB=False):
```

Comment:
How about reading this key `mockMongoDB` from the `msConfig` object? Then you can replace it when calling the super class as well.

Reply:
I was thinking about this possibility, too. I'll fix it.